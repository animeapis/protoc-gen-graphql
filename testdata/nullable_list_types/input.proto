syntax = "proto3";

package protoc_gen_graphql.test.wrappers;

message WithEmptyList {
  repeated string canBeNullable = 1;
}
