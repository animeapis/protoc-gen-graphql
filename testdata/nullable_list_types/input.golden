# DO NOT EDIT! Generated by protoc-gen-graphql.

type ProtocGenGraphqlTestWrappers_WithEmptyList {
  canbenullable: [String!]
}
